packages/compiler/src/render3/view/i18n/util.ts

```diff
@@ -50,6 +50,11 @@ export function hasI18nMeta(node: t.Node&{i18n?: i18n.I18nMeta}): boolean {
   return !!node.i18n;
 }
 
+export function isBoundI18nAttribute(node: t.TextAttribute|
+                                     t.BoundAttribute): node is t.BoundAttribute {
+  return node.i18n != null && !(node instanceof t.TextAttribute);
```

Comment:
May be invert this condition and check for `t.BoundAttribute`?
```suggestion
  return node.i18n != null && (node instanceof t.BoundAttribute);
```